src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx

```diff
@@ -78,13 +78,19 @@ const GeneralInfoStep = ({
     service: getUserById,
     defaultResponse: { firstName: '', lastName: '' }
   })
-  const { response: countries } = useAxios({
+
+  const {
+    loading: loadingCountries,
+    response: countries,
+    fetchData: getCountriesFetch
```

Comment:
i guess, you should not use both get and fetch in function name

Reply:
You are right. I fixed it.